web/src/app/page.tsx

```diff
@@ -10,7 +10,7 @@ export default function Home() {
 	const router = useRouter();
 
 	return (
-		<main className='flex justify-center my-10 '>
```

Comment:
why did you change this to a div?

Reply:
Because the main is now in the layout.tsx

Reply:
I dont see main in layout.tsx